src/NodaTime/LocalDateTime.cs

```diff
@@ -869,6 +869,18 @@ public ZonedDateTime InZone([NotNull] DateTimeZone zone, [NotNull] ZoneLocalMapp
         }
 
         /// <summary>
+        /// Deconstruct this <see cref="LocalDate"/> into its components.
+        /// </summary>
+        /// <param name="date">The date portion of the value.</param>
+        /// <param name="time">The time portion of the value</param>
```

Comment:
Either add a period here, or remove it from the one above. I don't mind which - I'm not fussy about being consistent *between* methods - but it's good to be consistent *within* a single method :)